starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,73 @@
+const express = require("express");
+const router = express.Router();
+const dbHelper = require("../dbHelper");
+const Celebrity = require("../models/celebrity.model");
+
+router.get("/celebrities", async (req, res, next) => {
+  try {
+    const celebrities = await dbHelper.viewCelebrities();
+    res.render("../views/celebrities/index.hbs", { celebrities });
+  } catch (error) {
+    next();
+    return error;
+  }
+});
+
+router.get("/celebrities/new", async (req, res, next) => {
+  res.render("../views/celebrities/new.hbs");
```

Comment:
When rendering views, since you have defined the "views" as the default views folder, you don't have to put a relative path to render your views.
You can ommit the ".hbs" aswell since hbs is defined as the default view engine aswell in the config.
`res.render("../views/celebrities/index.hbs", { celebrities });`
can become:
`res.render("celebrities/index", { celebrities });`